financials/Funding request template.txt

```diff
@@ -0,0 +1,24 @@
+## Template for funding request
+This funding request template was made based on past experiences and debates on the Grin Forum about the necessity to have clear, 

```

Comment:
This motivation is useful, but the instructions are also not crucial for the template or request itself. So you may want to consider keeping this section commented out in code, like so:

```
<!-- This is a multiline comment,
which is only visible during editing,
but not in the final output. -->
```